cgi-bin/dashboard-svn.cgi

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl -T
+# ${license-info}
+# ${developer-info
+# ${author-info}
+# ${build-info}
+
+use CGI;
+
+my $repo = "file:///var/www/svn/scdb";
```

Comment:
Do you have a place to suggest where to put it ?
AFAIU, there is no SVN repository definition in quattor itself